cypress/e2e/ui/menu.cy.js

```diff
@@ -25,7 +25,8 @@ describe('Menu', () => {
       });
 
       it('Utilization', () => {
-        cy.menu('Overview', 'Utilization').expect_explorer_title('Enterprise');
+        cy.menu('Overview', 'Utilization').expect_explorer_title('Enterprise'); // Need to have simulate_queue_worker running to prevent this page from being stuck on loading (leads tot error in Firefox tests)
```

Comment:
wow that's really weird - especially if it's only happening with FF.  cc @agrare have you seen this before?

Reply:
Even in my own environment outside of cypress this page will be stuck on loading unless I run simulate_queue_worker. Not sure if this is just on my end or not

Reply:
Chrome seems to just be easier for running Cypress tests on. It can run the test to visit this page, find the title and end the test. Firefox seems to get stuck on this page even though it can read the title. Even if this test passes it will cause the next test to fail.

Also, overall Chrome doesn't seem to error like Firefox does when visiting a page, then visiting another page before the previous page is done loading.